src/CommonDevice.php

```diff
@@ -495,6 +495,9 @@ public function import(array $input)
                         ];
                         break;
                 }
+            } else if (preg_match('/^.+models_id/', $field)) {
```

Comment:
Maybe this `null` condition should be added on all fields.

Reply:
No, sometimes an item just misses some data, so we should update it, not create a new one.
I tried adding `null` on all fields and it broke a lot of tests.